appinventor/components-ios/src/RelativeLayout.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  RelativeLayout.swift
+//  AIComponentKit
+//
+//  Created by Cindy Bishop on 2/21/25.
+//  Copyright © 2025 Massachusetts Institute of Technology. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+open class RelativeLayout: Layout {
+    
+  private var layoutManager: UIView
+  private var componentsToAdd: [ViewComponent] = []
+  
+  private let NOT_VALID: Int = -1
+  
+  public init(preferredEmptyWidth: Int?, preferredEmptyHeight: Int?) {
+    if (preferredEmptyWidth == nil && preferredEmptyHeight != nil) ||
+       (preferredEmptyWidth != nil && preferredEmptyHeight == nil) {
+        fatalError("RelativeLayout - preferredEmptyWidth and preferredEmptyHeight must be either both nil or both not nil")
```

Comment:
Note that fatalError crashes the app, so we should see if there's another way to handle this situation. Alternatively, don't make the inputs nullable and then it won't be an issue.

Reply:
I think since HVArrangement doesn't allow nullable inputs, I can remove this for `RelativeLayout`?

Reply:
public init(preferredEmptyWidth: Int, preferredEmptyHeight: Int) {